src/authentication/AuthService.js

```diff
@@ -0,0 +1,31 @@
+import { Log, UserManager } from 'oidc-client';
+import {UserManagerMock} from "./UserManagerMock";
+
+export class AuthService {
+  userManager;
```

Comment:
I would declare userManager as a global variable (so out of AuthService)
const userManager = ...
And AuthService would be just a function to init userManager with the right instance depending of the context.